contracts/strategy/anchor/AnchorNonUSTStrategy.sol

```diff
@@ -230,40 +245,71 @@ contract AnchorNonUSTStrategy is AnchorBaseStrategy {
             );
     }
 
-    /**
-     * @return Underlying value of UST amount
-     */
-    function _estimateUstAmountInUnderlying(uint256 ustAmount)
+    function _getUstAndUnderlyingPrices()
         internal
         view
-        virtual
-        returns (uint256)
+        returns (uint256 ustPrice, uint256 underlyingPrice)
     {
         (
             uint80 ustRoundID,
-            int256 ustPrice,
+            int256 ustPriceInt,
```

Comment:
to avoid conflict of return variables' name.

Reply:
What if you just changed to explicit returns? 

```
return (uint256(ustPriceInt), uint256(underlyingPriceInt));
```

I actually like it better than the named returns

Reply:
@naps62 likes names in return variable i guess. ;)

Reply:
I'm not sure he has a preference xD I really don't care. Sometimes I get confused because I'm not sure which variables are being returned, but this is a simple enough function. What I don't like is you having to name something with a `Int` prefix when you could just do a simple explicit return :/

Reply:
I do like names yes, but not because it saves the `return` call. I actually prefer having both. Names for readability, and explicit returns for readability as well

With that in mind, I don't think using explicit returns is a good fix here. We can use them, but leaving the function return signature as `(uint256, unit256)` is extremely unclear

Reply:
True, you need to check the return to know which one is which. I can agree with that. I still don't like the suffix `Int` but ok :+1: